test/utils/DeployAllowlistProxy.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache 2.0
```

Comment:
If this contract is only needed by allow list tests, put it in test/allowlist

Reply:
This will is also used by the erc1155 test